src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -74,7 +74,18 @@ public void process(Objection objection, String httpRequestId)
 
         // TODO update status to CHIPS_SENT
 
-        // TODO OBJ-172 send internal email
+        // send internal email
+        try {
+            CompanyProfileApi companyProfile = this.companyProfileService.getCompanyProfile(objection.getCompanyNumber(), httpRequestId);
```

Comment:
Could put this line outside the try block as it is common to both emails

Reply:
Actually, you could make a private method called sendEmails(...) and have the company profile lookup and email sending / catching in there
And get the company profile once only, before sending any emails